app/pages/admin/products-benchmarks.tsx

```diff
@@ -132,6 +114,11 @@ class ProductsBenchmarks extends Component<Props> {
             <br />
           </Col>
         </Row>
+        <Row>
+          <Col>
+            <Button onClick={this.toggleShowCreateForm}>New Product</Button>
+          </Col>
+        </Row>
```

Comment:
See my previous comment about unnecessarily introducing further change to UI post-v1 user launch. We have plenty of opportunity to co-design change through an intentional team process with v2.

Our current admins are used to looking for this button in the top right. (and have given positive feedback in the recent user studies about the ease of finding it in that location :)

If you're looking for the location it 'should' be aligned to, it was right-aligned within the grey panel at the top of the screen which says "Manage Products" - so only the original button's vertical location needed adjusting.

Reply:
This one I made the change with @rajpersram 's advice, since he's had user feedback saying that people expected the button on the left instead, it was an easy change to reduce user confusion. I'm happy to put it anywhere else!